### ISSUE TITLE
Azure: OPENAI_MODEL vs OPENAI_MODEL_NAME?

### ISSUE THREAD

**Issue by trangdata** [biocypher/ChatGSE#33]
In `set_azure_mode`, do we want to change "OPENAI_MODEL" to "OPENAI_MODEL_NAME" to be consistent with [biochatter](https://github.com/biocypher/biochatter/blob/f1c25722507c63edbd6a4d7965209b0e78761d77/test/test_llm_connect.py#L95)?

https://github.com/biocypher/ChatGSE/blob/eb0a7c42180dc7591acc93fb854d368a331aa85e/app.py#L169-L173

**Comment by slobentanzer** [comment]
Absolutely, I probably overlooked that. Do you want to file a PR, or do you want me to do it?
